lib/src/lifecycle.rs

```diff
@@ -12,13 +24,17 @@ use crate::internal;
 /// have no effect. Antithesis will treat the first time any process called 
 /// this function as the moment that the setup was completed.
 pub fn setup_complete(details: &Value) {
```

Comment:
Do we want to generalize this to accept any `T: Serialize`, like what we now do for `dispatch_output`? Same may apply to the `send_event` function.